packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/resolution/ResolutionFeatureTest.java

```diff
@@ -91,8 +91,9 @@ public void beforeLegacy() {
   public void before() {
     mockProfileLow = mock(EncoderProfiles.class);
     EncoderProfiles mockProfile = mock(EncoderProfiles.class);
-    EncoderProfiles.VideoProfile mockVideoProfile = mock(EncoderProfiles.VideoProfile.class);
-    List<EncoderProfiles.VideoProfile> mockVideoProfilesList = List.of(mockVideoProfile);
+    List<EncoderProfiles.VideoProfile> mockVideoProfilesList =
+        new ArrayList<EncoderProfiles.VideoProfile>();
+    mockVideoProfilesList.add(null);
```

Comment:
The `EncoderProfiles.VideoProfile` class is final static so it cannot be mocked. It's not actually tested in this class so I just set it to null so that the tests run properly.